docs/configuration/network_filters/mongo_proxy_filter.rst

```diff
@@ -151,6 +179,14 @@ mongo.logging_enabled
   % of messages that will be logged. Defaults to 100. If less than 100, queries may be logged
   without replies, etc.
 
+mongo.fault.fixed_delay.percent
+  Probability of an elibible MongoDB operation to be affected by
```

Comment:
should there be a note if there is no entry in the config.

Reply:
`percent` is a required field, should be in the config.

Reply:
nope, not if fault object isn't there

Reply:
This runtime setting is taken into consideration only if `fault` object is specified for mongo_proxy filter.

I can put into the comments that this runtime setting is used only if `fault` configuration is defined for the proxy.

Reply:
typo elibible, "are no active faults"